soil_id/global_soil.py

```diff
@@ -72,20 +77,24 @@
 ##################################################################################################
 #                                 getSoilLocationBasedGlobal                                     #
 ##################################################################################################
-def getSoilLocationBasedGlobal(lon, lat, plot_id):
-    # Extract HWSD-WISE Data
-    # Note: Need to convert HWSD shp to gpkg file
-    wise_data = extract_WISE_data(
-        lon,
-        lat,
-        file_path=config.WISE_PATH,
-        layer_name=None,
-        buffer_size=0.5,
-    )
+def list_soils_global(lon, lat):
```

Comment:
issue: this should be called list_soils() to match the us version. this module is namespaced and doesn't need a prefix.